### PR TITLE
Open two PRs and report one NameOrReport

### PR DESCRIPTION
**Pull request by SijieMei**
Open two PRs
report one NameOrReport for nifi. The new test required java 21 or above to build.

### CODE REVIEW

**Comment by Ajitesh27** [comment]
Please go through the instructions to add Moved/Renamed status in README. The old line must have the status MovedOrRenamed along with the notes. You must then add a new line for the moved test with latest details.



FAQ
How to indicate that a test is renamed or moved to a different location?
If the test method body is the same between two versions (e.g., if in old_sha, some.test.name has the same test method body as some.other.test.name in new_sha), we consider the two different versions of the test to be the same test. For the row with the older sha, please change the Status to MovedOrRenamed and add [Notes](https://github.com/TestingResearchIllinois/idoft/tree/main#adding-notes) describing which version the test is found to be renamed/moved.

If the test method body is different, we consider the two different versions of the test to be two different tests. For the row with the older sha, please change the Status to Deleted and add [Notes](https://github.com/TestingResearchIllinois/idoft/tree/main#adding-notes) describing which version the test is found to be renamed/moved and how the test method body differs between the two versions.

For either case, please also add a new row for the newer sha and test name. Once the preceding changes are made, all future pull request updates should only be made to the row with the newer sha.

File: pr-data.csv [hunk 1]
```diff
@@ -4369,7 +4369,7 @@ https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9
...
 https://github.com/GoogleCloudPlatform/DataflowTemplates,cc17ed154d8c3c385ab3f5a3653a6e4c02539bfa,v1,com.google.cloud.teleport.templates.common.BigQueryConvertersTest.testFailsafeJsonToTableRowValidInput,ID,,,
 https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9fde28553a88f68e4b,.,com.google.cloud.teleport.templates.common.DatastoreConvertersTest.testCheckNoKeyAllInvalid,ID,,,
-https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9fde28553a88f68e4b,.,com.google.cloud.teleport.templates.common.DatastoreConvertersTest.testCheckNoKeyBothCorrectAndInvalid,ID,,,
+https://github.com/GoogleCloudPlatform/DataflowTemplates,0d01accc1cfa29b762a764bc226496bfde25165c,v1,com.google.cloud.teleport.templates.common.DatastoreConvertersTest.testCheckNoKeyBothCorrectAndInvalid,ID,Opened,https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1919,
```
Comment (darko-marinov) [inline pr-data.csv:4372]: Please do not change the commit SHA where the flaky was reported. What's the new SHA that you want to add? Maybe you can report in between that the test was moved. You could get +2 points for that (but need to find the commit where the test was moved).

Comment (SijieMei) [reply pr-data.csv]: I removed the change of this line and will figure out the correct format to create a new PR. Sorry for the confusion.

**Comment by darko-marinov** [comment]
Please squash your 4 commits into 1.

**Comment by SijieMei** [comment]
Hi Professor, I squashed my commits into one.

**Pull request merged**